examples/application/quickstart3.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: AttachedCluster
+metadata:
+  name: kurator-member1
+  namespace: default
+spec:
+  kubeconfig:
+    name: kurator-member1
+    key: kurator-member1.config
+---
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: AttachedCluster
+metadata:
+  name: kurator-member2
+  namespace: default
+spec:
+  kubeconfig:
+    name: kurator-member2
+    key: kurator-member2.config
```

Comment:
We donot need to duplicately create the AttachedCluster in the three examples

Reply:
The benefit of repetition is that users can test the examples in any order